docs/advanced-users-and-devs.md

```diff
@@ -0,0 +1,87 @@
+# For advanced users and developers
+
+## For developers
+
+*Developed by Arriven: (https://github.com/Arriven)*
+
+This is a simple distributed load generation client written in go.
+It is able to fetch simple json config from a local or remote location.
+The config describes which load generation jobs should be launched in parallel.
+I do not intend to copy or replace it but rather provide a simple open source option.
```

Comment:
```suggestion
There are other tools doing that. I do not intend to copy or replace them but rather provide a simple open source alternative so that users have more options
```
This was a leftover line from the time when it was just an idea inspired by disbalancer.com, this wording is probably better considering current situation